pkg/volume/rbd/rbd_util.go

```diff
@@ -101,7 +101,7 @@ func (util *RBDUtil) AttachDisk(rbd rbd) error {
 	// mount it
 	globalPDPath := rbd.manager.MakeGlobalPDName(rbd)
 	mountpoint, err := rbd.mounter.IsMountPoint(globalPDPath)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
yes, if directory is not there, that indicates it is the first time to create then mountpoint. the method should continue to run. the mountpoint is later created.